presto-native-execution/presto_cpp/presto_protocol/presto_protocol.yml

```diff
@@ -177,6 +181,7 @@ JavaClasses:
   - presto-hive-metastore/src/main/java/com/facebook/presto/hive/BucketFunctionType.java
   - presto-hive-common/src/main/java/com/facebook/presto/hive/CacheQuotaRequirement.java
   - presto-hive-common/src/main/java/com/facebook/presto/hive/CacheQuotaScope.java
+  - presto-hive-common/src/main/java/com/facebook/presto/hive/BaseHiveColumnHandle.java
```

Comment:
Is this still needed?

Reply:
Yes, this is still needed. ColumnType enum is part of this class and it is required to be included in the protocol.